fdbserver/ProxyCommitData.actor.h

```diff
@@ -249,6 +243,27 @@ struct ProxyCommitData {
 		latencyBandConfig = newLatencyBandConfig;
 	}
 
+	void updateSSTagCost(const UID& id, const TagSet& tagSet, MutationRef m, int cost){
+		if(ssTagCommitCost.count(id) == 0) {
+			ssTagCommitCost[id] = TransactionTagMap<TransactionCommitCostEstimation>();
+		}
+		for(auto& tag: tagSet) {
+			auto& costItem = ssTagCommitCost[id][tag];
+			if(m.isAtomicOp()) {
```

Comment:
As far as I can tell, there isn't any use of these that looks at them separately. Instead, they are all added together as the total writes and total cost. Are there plans to use them individually?